sokol_app.h

```diff
@@ -6698,37 +6736,49 @@ _SOKOL_PRIVATE void _sapp_win32_restore_console(void) {
 }
 
 _SOKOL_PRIVATE void _sapp_win32_init_dpi(void) {
-
+    
+    DECLARE_HANDLE(DPI_AWARENESS_CONTEXT_T);
```

Comment:
Should this be moved up into:

https://github.com/floooh/sokol/blob/b2f60754841c0bbfad47da86974e43e494f11bef/sokol_app.h#L2114-L2127